.github/workflows/testground-nightly.yml

```diff
@@ -0,0 +1,45 @@
+name: Run Testground Nightly
+on:
+  schedule:
+    # Run at 05:00 UTC every day
+    - cron:  '0 5 * * *'
+jobs:
+  run-testground-nightly:
+      runs-on: ubuntu-latest
+      container: iptestground/testground:edge
+      steps:
+      - uses: actions/setup-go@v3
+        with:
+          go-version: "^1.19.0"
+      # Get testground and our test plan code
+      - uses: actions/checkout@v3
+        with:
+         repository: 'statechannels/testground'
+         path: "code/testground"
+      - uses: actions/checkout@v3
+        with:
+          repository: 'statechannels/go-nitro-testground'
+          path: "code/go-nitro-testground"
+          ref: main
+
+      # Update our test plan so it uses the latest main of go-nitro
+      - name: Update Test Dependency
+        run: go get github.com/statechannels/go-nitro@main
+        working-directory: "code/go-nitro-testground"
+
+      - name: Import Test 
+        run:    testground plan import --from ./go-nitro-testground
+        working-directory: "code"
+
+      - name: Run 3 min Test 
+        run: | 
+         testground --endpoint=${{secrets.TG_SERVER_URL}}  run s --wait \
+         -tp=isNightly=true -p=go-nitro-testground -t=virtual-payment \
```

Comment:
Not super important but in your opinion does this nightly run count as "CI"?

Reply:
Since this test has a longer duration I thought it might make sense to keep it out of the CI metric data. That way we're comparing like to like. I don't feel too strongly about it though!